atlas/packages/app/src/shared/components/Placeholder/Placeholder.tsx

```diff
@@ -2,14 +2,16 @@ import styled from '@emotion/styled'
 import { colors } from '@/shared/theme'
 
 type PlaceholderProps = {
-  width?: string
-  height?: string
+  width?: string | number
+  height?: string | number
   rounded?: boolean
 }
 
+const getPropValue = (v: string | number) => (typeof v === 'string' ? v : `${v}px`)
```

Comment:
I'm not sure it will ever happen, but this wouldn't work with percentages.

Reply:
Well if you wanted to do percentages, you would need to pass the full string (e.g. `'100%"`) anyway so it will work the same way, all string will be passed through. Only numbers are interpreted as pixels